docs/developer-docs/latest/update-migration-guides/migration-guides/v4/snippets/update-dependencies-snippet.md

```diff
@@ -0,0 +1,62 @@
+:::prerequisites
+Stop the server before starting the upgrade. 
+:::
+
+Upgrade all of the Strapi packages in the `package.json` to `4.0.6`:
+
+```jsx
+// path: ./package.json
```

Comment:
```suggestion
// path: package.json
```

`package.json` is at the root of the folder.
(ℹ️ `/` would imply you go into a (sub)folder).